### DISCUSSION TITLE
建议:在底部增加一个备案号文案

### DISCUSSION THREAD

**Discussion by jdnotes** [gaowei-space/markdown-blog#32] [Ideas]
大佬，能否在网站下面添加一个写备案号的底部栏，我部署了网站，但提示说 网站首页底部未添加正确备案号

**Comment by gaowei-space** [comment]
> 大佬，能否在网站下面添加一个写备案号的底部栏，我部署了网站，但提示说 网站首页底部未添加正确备案号

get

**Reply by jdnotes** [reply]
好的，大佬，很期待你的作品更新
